doc/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -8,13 +21,13 @@ In this section, we will walk through configuring the 3D sensors on your robot w
 * The Depth Image Occupancy Map Updater: which can take as input Depth Images (``sensor_msgs/Image``)
 
 YAML Configuration file (Point Cloud)
--------------------------------------
++++++++++++++++++++++++++++++++++++++
 
 We will have to generate a YAML configuration file for configuring the 3D sensors. Please see `this example file <https://github.com/ros-planning/panda_moveit_config/blob/master/config/sensors_kinect_pointcloud.yaml>`_ for processing point clouds, located in the `panda_moveit_config repository for Kinetic <https://github.com/ros-planning/panda_moveit_config>`_: ::
```

Comment:
Tell the user where to save this file and what to name it.

Reply:
Isn't this covered in the section below on [update the launch files](https://github.com/ros-planning/moveit_tutorials/pull/178/files/980d0bd66ce9dc22fa733e76b740ec2947f5d389#diff-6b6658aa38668a27c691b7b7a800f585R99)?